ingest-app/src/cmr/ingest/api/bulk.clj

```diff
@@ -38,13 +38,17 @@
   (generate-xml-status-list result status-list-key status-key id-key nil))
  ([result status-list-key status-key id-key created-at-key]
   (generate-xml-status-list result status-list-key status-key id-key created-at-key nil))
- ([result status-list-key status-key id-key created-at-key additional-keys]
+ ([result status-list-key status-key id-key created-at-key name-key]
+  (generate-xml-status-list result status-list-key status-key id-key created-at-key name-key nil))
+ ([result status-list-key status-key id-key created-at-key name-key additional-keys]
   (xml/element status-list-key {}
     (for [status (get result status-list-key)
           :let [message (:status-message status)]]
      (xml/element status-key {}
       (when created-at-key
         (xml/element created-at-key {} (str (:created-at status))))
+      (when name-key 
```

Comment:
I think we will always want to return a name. If there is no name we should return some generated default.